postgres/pom.xml

```diff
@@ -55,6 +55,11 @@
             <artifactId>postgresql</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs-annotations</artifactId>
```

Comment:
Can we make sure this is managed to `<scope>provided</scope>` so we don't export this dependency?